### PR TITLE
Add Identify keyboard shortcut to play observation's sound

### PR DESCRIPTION
**Pull request by grfeller**
[#3419](https://github.com/inaturalist/inaturalist/issues/3419)

Set focus to first `<audio>` element and play associated sound file. If an `<audio>` element is already active, the `togglePlayFirstSound` action exits early to avoid interfering with default processing.

### CODE REVIEW

File: app/webpack/observations/identify/components/observation_modal.jsx [hunk 1]
```diff
@@ -391,6 +391,7 @@ class ObservationModal extends React.Component {
       { keys: ["i"], label: I18n.t( "add_id" ) },
       { keys: ["f"], label: I18n.t( "add_to_favorites" ) },
       { keys: ["z"], label: I18n.t( "zoom_photo" ) },
+      { keys: ["Space"], label: I18n.t( "play_first_sound" ) },
```
Comment (kueda) [inline app/webpack/observations/identify/components/observation_modal.jsx:394]: Please make "Space" all-caps to match the other named keys like SHIFT.

File: app/webpack/observations/identify/actions/current_observation_actions.js [hunk 2]
```diff
@@ -1126,6 +1126,27 @@ export function subscribe( ) {
...
+    player.focus();
+    if ( player.paused ) {
+      player.play( );
+    }
+    else {
+      player.pause( );
+    }
+  };
+}
+
```
Comment (kueda) [inline app/webpack/observations/identify/actions/current_observation_actions.js:1149]: Please abide by the eslint rules, at least for the code that you're adding. You can run `npx eslint --fix app/webpack/observations/identify/actions/current_observation_actions.js` if your editor doesn't automatically show you the eslint errors.

**Review by kueda — CHANGES_REQUESTED** [review]
Works well in Firefox. In addition to the inline comments, here are some bugs / requests

- [ ] In Safari, SPACE will play the sound and focus on the player, but hitting SPACE again will not pause; seems like maybe SPACE does nothing while the player has focus, but only in Safari

I guess that was just one bug, singular. If you can find a workaround for Safari, great, but I think this is a step forward even if that seems impossible.

**Comment by grfeller** [comment]
Firefox, Chromium and Safari should now behave as described (at least on macOS 12 and Ubuntu 20.04):
- SPACE will play the sound and focus on the *first* player. Hitting SPACE again will pause. 
- To avoid confusion, I removed the processing for subsequent `<audio>` elements; this makes the behavior *Play first sound* rather than *Play first / current sound* as previously implemented.

**Implementation**
- Binding to `keydown` instead of `keyup` suppressed the Firefox [shortcut for Toggle Play / Pause](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_media-shortcuts),

**Review by kueda — APPROVED** [review]

**Comment by kueda** [comment]
Thanks, @grfeller! Sorry it took me a while to get back to this.

**Comment by grfeller** [comment]
@kueda thanks for your help and feedback in making this PR successful.

**Pull request merged**